CSD412webProject/CSD412webProject/Models/User.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Microsoft.AspNetCore.SignalR;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace CSD412webProject.Models
+{
+    public class User
+    {
+
+        public int Id { get; set; }
+        public string Name { get; set; }
+        public string LastName { get; set; }
+        public string Email { get; set; }
+        public DateTime DateOfBirth => DateOfBirth.Date;
+        public string Role { get; set; }
+
+        public User()
+        {
```

Comment:
I would put here also ID that getting auto incremental static int IdCounter (this.Id = IdCounter++;